### DISCUSSION TITLE
Updating bat using dpkg

### DISCUSSION THREAD

**Discussion by navneethc** [sharkdp/bat#2326] [General]
Ubuntu 20.04

I have v0.21 installed via dpkg and I'm trying to update it to 0.22.1. When I run `dpkg` the command exits saying that a previous version exists. Should I always uninstall a previous version before installing a new one? Does `bat` not support automatic updates?

```
❯ sudo dpkg -i bat-musl_0.22.1_amd64.deb
[sudo] password for navneethc:
Selecting previously unselected package bat-musl.
dpkg: regarding bat-musl_0.22.1_amd64.deb containing bat-musl:
 bat-musl conflicts with bat
  bat (version 0.21.0) is present and installed.
  bat provides bat and is present and installed.

dpkg: error processing archive bat-musl_0.22.1_amd64.deb (--install):
 conflicting packages - not installing bat-musl
Errors were encountered while processing:
 bat-musl_0.22.1_amd64.deb
~
❯
```

**Comment by keith-hall** [comment]
`bat` and `bat-musl` are two different packages and they conflict with each other. Stick to one or the other and they should be upgradable.

**Reply by navneethc** [reply]
🤦‍♂️ 

I just download the first deb package I saw, not realising that I had to expand the list to get the actual one I wanted.

Thanks for your help.
